test/automation/src/application.ts

```diff
@@ -55,12 +54,12 @@ export class Application {
 		return this._workspacePathOrFolder;
 	}
 
-	get extensionsPath(): string {
+	get extensionsPath(): string | undefined {
 		return this.options.extensionsPath;
 	}
 
-	private _userDataPath: string;
-	get userDataPath(): string {
+	private _userDataPath: string | undefined;
+	get userDataPath(): string | undefined {
```

Comment:
I opted for undefined rather than using the default value just because I didn't want to be in the business of keeping that in sync